src/components/AddTodoForm/AddTodoForm.tsx

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import users from '../../api/users';
```

Comment:
The `users` data is directly imported here, which violates the requirement to prepare data in the `App` component and pass it down as props. Consider refactoring to pass `users` as a prop from the `App` component.